prototypes/form-wizard/confirmation.html

```diff
@@ -1,5 +1,7 @@
 ---
 layout: page
+permalink: /confirmation/
+title: Thank you for your submission
```

Comment:
Given the feedback we've heard, maybe we want to get rid of the nav all together.

[Navbar code is here](https://github.com/18F/foia-recommendations/blob/master/prototypes/form-wizard/_includes/header.html#L30), you can see it's conditioned on title, we can add a property like `show_in_nav` to the frontmatter and condition off that for the nav.